### ISSUE TITLE
npm install fails

### ISSUE THREAD

**Issue by debris** [nikku/karma-browserify#118]
I've just noticed that my travis build fails on karma-browserify install.
Travis link: https://travis-ci.org/ethereum/ethereum.js/jobs/56538523

```
$ npm install 
npm ERR! Linux 2.6.32-042stab090.5
npm ERR! argv "/home/travis/.nvm/versions/node/v0.12.1/bin/node" "/home/travis/.nvm/versions/node/v0.12.1/bin/npm" "install"
npm ERR! node v0.12.1
npm ERR! npm  v2.5.1
npm ERR! shasum check failed for /tmp/npm-2426-064279b2/registry.npmjs.org/karma-browserify/-/karma-browserify-4.1.1.tgz
npm ERR! Expected: fe3e43f2eab1934ee08dc47a89575d52cb0bb77b
npm ERR! Actual:   0a9fc7dc0e9aca0d4346067a2e61aee57afdadf1
npm ERR! From:     https://registry.npmjs.org/karma-browserify/-/karma-browserify-4.1.1.tgz
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <http://github.com/npm/npm/issues>
npm ERR! Please include the following file with any support request:
npm ERR!     /home/travis/build/ethereum/ethereum.js/npm-debug.log
The command "eval npm install " failed. Retrying, 2 of 3.
```

**Comment by peter-mouland** [comment]
+1
i've temporarily pointed my package.json to `nikku/karma-browserify`

**Comment by nikku** [comment]
Looks like the npm publish operation screwed up. I published an identical v4.1.2 which installs correctly.

Feel free to reopen this issue if you continue to see issues.
